Sources/VRMKit/VRM/Node.swift

```diff
@@ -51,5 +51,75 @@ extension GLTF {
             case extensions
             case extras
         }
+
+        public struct NodeExtensions: Codable {
+            public let nodeConstraint: NodeConstraint?
+
+            private enum CodingKeys: String, CodingKey {
+                case nodeConstraint = "VRMC_node_constraint"
+            }
+
+            public struct NodeConstraint: Codable {
+                public let specVersion: String
+                public let constraint: Constraint
+                public let extensions: CodableAny?
+                public let extras: CodableAny?
+
+                public struct Constraint: Codable {
+                    public let roll: RollConstraint?
+                    public let aim: AimConstraint?
+                    public let rotation: RotationConstraint?
+                    public let extensions: CodableAny?
+                    public let extras: CodableAny?
+
+                    public struct RollConstraint: Codable {
+                        public let source: Int
+                        public let rollAxis: RollAxis
+                        public let weight: Double?
+                        public let extensions: CodableAny?
+                        public let extras: CodableAny?
+
+                        public enum RollAxis: String, Codable {
+                            case X
+                            case Y
+                            case Z
```

Comment:
他の部分も命名規則をSwiftにあわせているので、こちらもこのようにしたいです
```suggestion
                            case x = "X"
                            case y = "Y"
                            case z = "Z"
```

Reply:
命名規則を合わせるため、修正してプッシュしました。